Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -1402,6 +1402,19 @@ bool Element::is_actually_disabled() const
     return false;
 }
 
+// https://w3c.github.io/DOM-Parsing/#dom-element-outerhtml
+WebIDL::ExceptionOr<String> Element::outer_html() const
+{
+    return serialize_fragment(DOMParsing::RequireWellFormed::Yes, FragmentSerializationMode::Outer);
+}
+
+// https://w3c.github.io/DOM-Parsing/#dom-element-outerhtml
+WebIDL::ExceptionOr<void> set_outer_html(String const&)
```

Comment:
Missing `Element::`, surprised there isn't a linker error from the generated prototypes

Reply:
:man_facepalming: